index.ios.js

```diff
@@ -1,77 +0,0 @@
-/**
- * @providesModule MusicControl
- * @flow
- */
-'use strict';
-
-import { NativeModules, NativeEventEmitter } from 'react-native';
-const NativeMusicControl = NativeModules.MusicControlManager;
-
-/**
- * High-level docs for the MusicControl iOS API can be written here.
- */
-var handlers = { };
-var subscription = null;
-
-var MusicControl = {
-
-  STATE_PLAYING: NativeMusicControl.STATE_PLAYING,
-  STATE_PAUSED: NativeMusicControl.STATE_PAUSED,
-  STATE_ERROR: NativeMusicControl.STATE_PAUSED,
-  STATE_STOPPED: NativeMusicControl.STATE_PAUSED,
-  STATE_BUFFERING: NativeMusicControl.STATE_PAUSED,
```

Comment:
Oh, you are right, but I think the better solution is to use a Platform switch in the same file instead of copying everything to a new file and just changing the constants.